django_prbac/admin.py

```diff
@@ -57,25 +60,16 @@ def parameters__csv(self, instance):
         'description',
     ]
 
+
 class GrantAdmin(admin.ModelAdmin):
 
     model = Grant
 
-    def edit_link(self, instance):
```

Comment:
We don't need this (why this was present in the original code )

Reply:
I think fine to remove it as long as all tests still pass.

Reply:
Oh, this appears to be part of the admin interface. I think it should not be removed.

Reply:
Yeh, but it's redundant because Django admin allows the same functionality right from box

Reply:
Ah, I didn't realize that. In that case, fine to remove.